faq.yaml

```diff
@@ -51,7 +54,7 @@ Basics:
         <br/>
         <br/> A message should be returned by Rythm stating its current prefix.
     Can I play a playlist with Rythm?: |
-        <br/> Rythm can reproduce playlists from a variety of online audio sources including youtube and soundcloud.
+        <br/> Rythm can reproduce playlists from a variety of online audio sources including YouTube and SoundCloud.
```

Comment:
Would "Rythm can use playlists..." be better for here?

Reply:
Agreed